vagrant/provisioning/roles/httpd/templates/index.html

```diff
@@ -4,7 +4,7 @@
   </head>
   <body>
     <p>ArkCase webapp is here: <a href="/arkcase">ArkCase CE</a></p>
-    <p>ArkCase administrator account: arkcase-admin@arkcase.org, password @rKc@3e</p>
+    <p>ArkCase administrator account: {{ admin_user }}@arkcase.org, password @rKc@3e</p>
```

Comment:
The whole PR is already done. Not sure why Github shows the old version of file on develop branch.
This one here should be added. Use {{ arkcase_admin_user }}@{{ ldap_user_domain }}